agent/src/softwarecontaineragent.cpp

```diff
@@ -241,23 +247,52 @@ void SoftwareContainerAgent::setContainerName(ContainerID containerID, const std
     }
 }
 
-void SoftwareContainerAgent::shutdownContainer(ContainerID containerID)
+bool SoftwareContainerAgent::shutdownContainer(ContainerID containerID)
 {
-    shutdownContainer(containerID, m_softwarecontainerWorkspace->m_containerShutdownTimeout);
+    return shutdownContainer(containerID, m_softwarecontainerWorkspace->m_containerShutdownTimeout);
 }
 
-void SoftwareContainerAgent::shutdownContainer(ContainerID containerID, unsigned int timeout)
+bool SoftwareContainerAgent::shutdownContainer(ContainerID containerID, unsigned int timeout)
 {
     profilefunction("shutdownContainerFunction");
     if (m_shutdownContainers) {
```

Comment:
Negating this if clause and putting a return inside would reduce the indentation